src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,123 @@
+- **Logical data replication (LDR)** (Preview): A cross-cluster replication tool between multiple active CockroachDB clusters, which supports a range of topologies. LDR provides eventually consistent, table-level replication.
+
+### Choose an HA strategy
+
+CockroachDB uses synchronous, built-in Raft replication to create and distribute copies of data, ensuring consistency across the data copies. The database can tolerate nodes going offline without service interruption, whether in a single-region or [multi-region]({% link {{ page.version.version }}/multiregion-overview.md %}) cluster.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Single-region replication (synchronous)</th>
+    <th>Multi-region replication (synchronous)</th>
+    <th>Logical data replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>0 seconds</td>
+    <td>0 seconds</td>
+    <td>Immediate mode 0.5 seconds<br>Transactional mode ~10s of seconds</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Add application cutover time</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>Region-local write latency<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+    <td>Cross-region write latency<br>p50 latency > 50ms  (for a multi-region cluster in us-east1, us-east-2, us-west-1)</td>
+    <td>Region-local latency depending on design<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Automatic</td>
+    <td>Automatic</td>
+    <td>Semi-automatic</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x<br>Networking between availability zones</td>
+    <td>1x<br>Networking between datacenters</td>
+    <td>2x<br>Networking between availability zones or datacenters</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Minimum regions</b>
+    </td>
+    <td>1</td>
+    <td>3</td>
+    <td>2</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Fault tolerance</b>
+    </td>
+    <td>Zero RPO node, availability zone failures</td>
+    <td>Zero RPO node, availability zone failures, region failures</td>
+    <td>Zero RPO node, availability zone, region failures with loss up to RPO</td>
```

Comment:
this makes it a bit confusing, like all the nodes, AZs, and region failures are zero RPO. Are we able to elaborate on how you could achieve this like ('zero rpo node, availability zone within a cluster', region failures with loss up to RPO  in a two-region (or 2dc) setup

Reply:
I've updated as we discussed in our 1:1, which is the fault tolerance row of both the LDR and PCR columns. And then added "within a cluster" to both the sr and mr columns